views/pages/passwordreset.pug

```diff
@@ -34,19 +36,19 @@ append scripts
                     },
                     success: (response, textStatus, xhr) => {
                         $("#submit-btn").prop("disabled",false);
-                        showToast("A password reset link was sent. Click the link in the email to create a new password.");
+                        toast.show("A password reset link was sent. Click the link in the email to create a new password.");
```

Comment:
@Enubia hey, how come some of them are just "toast.show()" and some are "new DG.Toast.show()" in this file?

Reply:
because some are instantiated and assigned to a variable and some aren't, depends on if it's needed in more places

Reply:
gotcha that makes sense!